tests/entrypoints/openai/test_lora_lineage.py

```diff
@@ -81,3 +84,26 @@ async def test_check_lora_lineage(client_for_lora_lineage: openai.AsyncOpenAI,
     assert all(lora_model.parent == MODEL_NAME for lora_model in lora_models)
     assert lora_models[0].id == "zephyr-lora"
     assert lora_models[1].id == "zephyr-lora2"
+
+
+@pytest.mark.asyncio
+async def test_dynamic_lora_lineage(
```

Comment:
Note for reviewers: This was the missing test case that covers dynamically loaded adapters showing up in the response from `/v1/models`